pileface.py

```diff
@@ -1,16 +0,0 @@
-import random
-
-def pileFace():
-    PF=["pile","face"]
-    inputUser=eval(input("pile ou face ? met 0 pour pile ou 1 pour face: ")) #demande à user pile ou face
```

Comment:
Eval should be avoided at all costs : it is a security issue. Someone can write whichever code they want and it will run on your machine.
Instead you can use `int()` to convert a string to a number